lib/api/apiUtils/object/getReplicationInfo.js

```diff
@@ -1,5 +1,44 @@
 const s3config = require('../../../Config').config;
 
+function _getReplicationInfo(rule, replicationConfig, content, operationType) {
+    const storageClasses = rule.storageClass ?
+        rule.storageClass.split(',') : [];
+    const storageTypes = [];
+    const backends = [];
+    storageClasses.forEach(storageClass => {
+        const location = s3config.locationConstraints[storageClass];
+        if (location && ['aws_s3', 'azure'].includes(location.type)) {
+            storageTypes.push(location.type);
+        }
+        backends.push({
+            site: storageClass,
+            status: 'PENDING',
+        });
+    });
+    if (storageTypes.length > 0 && operationType) {
+        content.push(operationType);
+    }
+    // If no StorageClass, we replicate to the sole Scality S3 destination.
+    if (backends.length === 0) {
+        const dest = s3config.replicationEndpoints &&
+            s3config.replicationEndpoints.find(site =>
+            site.servers !== undefined);
+        backends.push({
+            site: dest && dest.site,
```

Comment:
Maybe I'm understanding the architecture all wrong, but won't this set `site` to a boolean instead of (what I'm assuming should be the default) `zenko`?

Reply:
It's just JavaScript weirdness. It will return the value of `dest.site`.

Reply:
Huh, cool. You're a srs Javascript wizard :)